torchao/dtypes/floatx/float8_layout.py

```diff
@@ -333,13 +333,12 @@ def _linear_fp8_act_fp8_weight_impl(
     input_scale = input_tensor.tensor_impl.scale
     # Handle case where input tensor is more than 2D
     inpt_data = inpt_data.reshape(-1, inpt_data.shape[-1])
-
     # Handle rowwise case
     if _is_rowwise_scaled(weight_tensor):
         assert _is_rowwise_scaled(input_tensor), (
             "Input tensor must be rowwise block size"
         )
-        w_scale = w_scale.unsqueeze(-1).T
```

Comment:
Why was this necessary? Is this intentional?

Reply:
TBH I am not entirely sure what happend to all this code but rowwise scales was  stored in shape (N_rows) shape